int32 latitude
int32 longitude
---
bool response